lib/Horde/Imap/Client/Url/Base.php

```diff
@@ -170,4 +170,19 @@ public function unserialize($data)
         $this->_parse($data);
     }
 
+    /**
+     * @return array
+     */
+    public function __serialize()
```

Comment:
same here. the old one serialized to a string, now need to use an array of string